packages/extra/src/MathUtil.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * @copyright (c) 2016, Philipp Thürwächter & Pattrick Hüper
+ * @copyright (c) 2007-present, Stephen Colebourne & Michael Nascimento Santos
+ * @license BSD-3-Clause (see LICENSE in the root directory of this source tree)
+ */
+import { ArithmeticException } from '@js-joda/core';
+
+export const MAX_SAFE_INTEGER = 9007199254740991;
+export const MIN_SAFE_INTEGER = -9007199254740991;
+
+/**
+ * Math helper with static function for integer operations
+ */
+export class MathUtil {
```

Comment:
hmmm... i dont like duplicating MathUtil.js ... i wonder wether we cant somehow import it from js-joda/core ... and yes, i now it isn't exported by default ... but duplicating seems wrong ... maybe we need a utils package or sth similar?

Reply:
I don't like duplicating MathUtil.js too, but this also applies to `requireInstance`, `requireNonNull` and test.js. Currently there is nothing leaking into public API here, so how about refactoring this later?
Importing from js-joda/core doesn't seem to be possible. I would be happy to see some utils package(s) instead.

Reply:
good point as well ... let's leave it for later and use it as it is right now ...

Reply:
Its possible to use MathUtil from the core package, see https://github.com/js-joda/js-joda/blob/328c57028f43aa078b0329e2bb104bcc98aa61f1/packages/locale/src/format/parser/LocalizedOffsetPrinterParser.js#L14 for an example. Could you try to use this one instead ?